### ISSUE TITLE
Extension to non-native ODE solvers?

### ISSUE THREAD

**Issue by NirvikNU** [JuliaDiffEq/DiffEqUncertainty.jl#9]
Can this package be used for non-native solvers like that of Sundials, ODEInterface etc.?

**Comment by ChrisRackauckas** [comment]
No, that's pretty fundamental to the method and what's exposed from those solvers. At least it could never do the more accurate version which uses the adaptive error estimates
